playbooks/roles/deploy-acid/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Install packages
+  become: yes
+  apt:
+    name: "{{ item }}"
+    update_cache: yes
+  with_items:
+    - supervisor
+    - python3-venv
+
+- name: Clone ACID repository
+  git:
+    repo: 'https://github.com/codilime/acid.git'
+    dest: "{{ ansible_env.HOME }}/acid"
+    force: yes
+
+- name: Create virtualenv and install python package
+  shell: |
+    python3.6 -m venv .venv; \
+    . .venv/bin/activate; \
+    pip install --upgrade pip; \
+    pip install setuptools wheel; \
+    pip install -r requirements.txt; \
+    pip install gunicorn
+  args:
+    chdir: "{{ ansible_env.HOME }}/acid"
+
+- name: Copy additional files
+  copy:
+    src: "{{ item }}"
+    dest: "{{ ansible_env.HOME }}/acid/"
+  with_items: "{{ files_to_copy }}"
+
+- name: Setup sueprvisor configuration
+  template:
+    src: templates/supervisor_acid.conf
+    dest: /etc/supervisor/conf.d/acid.conf
+  become: yes
```

Comment:
Put it always as first key for better readability. Everybody expects this key at the top of task. "Become" creates very important context for a task.